src/MeshManager.cpp

```diff
@@ -285,6 +341,10 @@ namespace blitzdg {
         return Dim;
     }
 
+    index_vector_type& MeshManager::get_BCType() const {
```

Comment:
In general, small functions that just return a value or reference are better defined within the class declaration since this causes them to be inlined. Alternatively, you can define these member functions outside the class declaration and mark them inline at the point of definition.

Reply:
Okay, I'll save this for a wider refactor though.